packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -480,17 +478,34 @@ function CellWithScheduleIcon({ scheduleId, children }) {
     color: 'inherit',
   };
 
+  let recurring = schedule && schedule._date && !!schedule._date.frequency;
+  let onScheduleIconClick = () => onNavigateToSchedule(scheduleId);
+  let ScheduledIcon = () =>
+    recurring ? (
+      <ArrowsSynchronize style={style} onClick={onScheduleIconClick} />
+    ) : (
+      <CalendarIcon
+        style={{ ...style, transform: 'translateY(-1px)' }}
+        onClick={onScheduleIconClick}
+      />
+    );
+
+  let onTransferIconClick = () =>
+    !isTemporaryId(transaction.id) &&
+    onNavigateToTransferAccount(transferAccount.id);
+  let TransferDirectionIcon = () =>
+    (transaction._inverse ? -1 : 1) * transaction.amount > 0 ? (
+      <LeftArrow2 style={style} onClick={onTransferIconClick} />
+    ) : (
+      <RightArrow2 style={style} onClick={onTransferIconClick} />
```

Comment:
Can you make all of these into `<Button bare ...>` to make it clear on hover that you can click? (it might be difficult to get it to fit in layout wise, you may need to use negative margins to make the rows stay the same height as they are now)

Reply:
Updated :)